packages/nimble-components/src/anchor-tabs/tests/anchor-tabs.spec.ts

```diff
@@ -66,10 +67,10 @@ describe('AnchorTabs', () => {
             expect(element.tablist.getAttribute('role')).toBe('tablist');
         });
 
-        it('should have a slots named "start", "anchortab", and "end", in that order', () => {
+        it('should have a slots named "start", "tab", and "end", in that order', () => {
             const slots = element.shadowRoot?.querySelectorAll('slot');
             expect(slots![0]?.getAttribute('name')).toBe('start');
-            expect(slots![1]?.getAttribute('name')).toBe('anchortab');
+            expect(slots![1]?.getAttribute('name')).toBe('tab');
```

Comment:
I'm probably missing something, but wouldn't this value come from `tabSlotName`, which has a value of `"anchortab"` for the anchor tabs?

Reply:
Nope, I just missed updating this.